crates/net/discv5/src/lib.rs

```diff
@@ -242,13 +242,24 @@ impl Discv5 {
                 // peer has been discovered as part of query, or, by incoming session (peer has
                 // discovered us)
 
-                self.metrics.discovered_peers_advertised_networks.increment_once_by_network_type(&enr);
-
                 self.metrics.discovered_peers.increment_established_sessions_raw(1);
 
                 self.on_discovered_peer(&enr, remote_socket)
             }
-            _ => None,
+            discv5::Event::UnverifiableEnr {
+                enr,
+                socket,
+                node_id: _,
+            } => {
+                // covers `reth_discv4::DiscoveryUpdate` equivalents `DiscoveryUpdate::Added(_)`
+                // and `DiscoveryUpdate::DiscoveredAtCapacity(_)
```

Comment:
unclear what the references to the types mean here

please clarify what covers + equivalents means here